expo/.cache/v/cache/lastfailed

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
is there a way to exclude this from git?  I've been manually removing it every checkin from the check list, but second I miss one... - I tried adding the path to git ignore, but it doesn't work.  Is there a place to configure where this cache shows up?  It started appearing with py.test